samples/Common.c

```diff
@@ -693,6 +693,7 @@ STATUS createSampleConfiguration(PCHAR channelName, SIGNALING_CHANNEL_ROLE_TYPE
 
 #ifdef IOT_CORE_ENABLE_CREDENTIALS
     PCHAR pIotCoreCredentialEndPoint, pIotCoreCert, pIotCorePrivateKey, pIotCoreRoleAlias, pIotCoreThingName;
+    CHK_ERR((pIotCoreThingName = getenv(IOT_CORE_THING_NAME)) != NULL, STATUS_INVALID_OPERATION, "AWS_IOT_CORE_THING_NAME must be set");
```

Comment:
here we are checking that the iot thing name matches with the channel name but in the policy we are authz ing all channels isn't?

Reply:
Sorry for my ignorance, but I do not understand what you mentioned. I did not check the iot thing name matching with the channel name. Would you mind giving me more information?

Reply:
The codebase is using the macro versions - GETENV

Reply:
I changed all the getenv in the common.c. Thanks for your review. :)